src/core/CCArray.mli

```diff
@@ -70,7 +74,7 @@ val fold_map : ('acc -> 'a -> 'acc * 'b) -> 'acc -> 'a t -> 'acc * 'b t
 
 val scan_left : ('acc -> 'a -> 'acc) -> 'acc -> 'a t -> 'acc t
 (** [scan_left f acc a] returns the array
-    [ [|acc; f acc x0; f (f acc a.(0)) a.(1); …|] ].
+    [ [|acc; f acc x0; f (f acc a.(0)) a.(1); ...|] ].
```

Comment:
come on, is that really necessary? 😆

Reply:
55+ dotdotdot are already present in the code.
They must be modified?

```
Also, by trying 

$ ocamldoc -html CCArray.mli

I've got (with  CCArray.html):

 ... [|acc; f acc x0; f (f acc a.(0)) a.(1); â€¦|] 
```

```

Reply:
55+ dotdotdot are already used in the code.